static/css/index.css

```diff
@@ -1,7 +1,7 @@
 html,body {
   height: 100%;
   margin: 0;
-  #background-color: #e6eef4;
```

Comment:
were these changes needed for the prescreen?

Reply:
They weren't. I got rid of them.